client/client.go

```diff
@@ -33,4 +33,7 @@ type Client interface {
 
 	// KV returns access to the distributed configuration store
 	KV() (kv.Store, error)
+
+	// Txn returns access to the transaction store
+	Txn() (kv.TxnStore, error)
```

Comment:
Would it better to call it `TxnStore`? `Txn` feels like it should return a `transaction` object